runtime-modules/content/src/lib.rs

```diff
@@ -2348,6 +2353,39 @@ decl_module! {
             }
         }
 
+        /// Claims an accumulated channel reward for a council.
+        #[weight = 10_000_000] // TODO: adjust weight
+        pub fn claim_council_reward(
+            origin,
+            channel_id: T::ChannelId,
+        ) {
+            let channel = Self::ensure_channel_exists(&channel_id)?;
+
+            ensure_actor_authorized_to_claim_council_reward::<T>(origin, &channel.owner)?;
+
+            ensure!(
```

Comment:
Don't we have en explicit ensure for this? Seems this will be done again and again, so a more succinct one liner which can be repeated would be nice?

Reply:
Sure. Fixed: https://github.com/Joystream/joystream/pull/3562/commits/84782b5ed856271539b530963a5689424e322a03